ParisBNF/abb/BNFabb107.xml

```diff
@@ -0,0 +1,94 @@
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b52507214q">BnF Éthiopien d'Abbadie 107</idno>
+                    </msIdentifier>
+                    <history>
+                            <origin>
+                                <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was commissioned by <persName
+                                    ref="PRS1071dAbbadi" role="owner"/> in Ethiopia.  </origin>
+                            <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
+                                <date when="1902">1902</date>.</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">216-218</citedRange>
+                                            <citedRange unit="number">219</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Chaine1912Catalogue"/>
+                                            <citedRange unit="page">67</citedRange>
+                                            <citedRange unit="number">107</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">119-121</citedRange>
+                                            <citedRange unit="number">107</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="BetaEsraelLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-10-24">Created a record.</change>
```

Comment:
I would write "Created a stub" to make it clear that this is supposed to be developed to a record later

Reply:
Ok! I will change it!